packages/admin/admin-theme/src/componentsTheme/MuiAccordion.ts

```diff
@@ -0,0 +1,38 @@
+import { accordionDetailsClasses, accordionSummaryClasses } from "@mui/material";
+
+import { mergeOverrideStyles } from "../utils/mergeOverrideStyles";
+import { GetMuiComponentTheme } from "./getComponentsTheme";
+
+export const getMuiAccordion: GetMuiComponentTheme<"MuiAccordion"> = (component, { palette }) => ({
+    ...component,
+    defaultProps: {
+        square: true,
+        disableGutters: true,
+        elevation: 0,
+        ...component?.defaultProps,
+    },
+    styleOverrides: mergeOverrideStyles<"MuiAccordion">(component?.styleOverrides, {
+        root: {
+            border: `solid ${palette.divider}`,
+            "&:not(:last-child)": {
+                marginBottom: "20px",
+            },
+            "&:before": {
+                display: "none",
+            },
+            [`& .${accordionSummaryClasses.expandIconWrapper}.${accordionSummaryClasses.expanded}`]: {
+                transform: "rotate(90deg)",
+            },
+            [`& .${accordionSummaryClasses.root}`]: {
```

Comment:
@jamesricky Shouldn't summary and details styles be styled in their respective components themes?

Reply:
Yes, good point 👍🏼